src/tests/mod.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_use]
+pub(crate) mod common {
+    use crate as pyo3;
+    include!("../../tests/common.rs");
```

Comment:
I understand that it is additional churn, but I think having the common file placed at `src/tests/common.rs` and `include!`ing it using relative paths in the `tests` directory would be easier follow.

Reply:
Sure thing, I was wondering similarly 👍